### DISCUSSION TITLE
How to get the free energy value for FFS method

### DISCUSSION THREAD

**Discussion by zheng001a** [MICCoM/SSAGES-public#43] [General]
Dear SSAGES Developers,

I am a graduate student studying for a master's degree.
I am using the FFS method in SSAGES software to study droplet wetting behavior on rough surfaces and would like to be able to get the free energy values during droplet wetting.

Please allow me to ask some question.
Could you tell me how to output the free energy value for FFS method?
Alternatively, is there any other method or software that can help me get the free energy value for the FFS method?

I would appreciate if you answer my question.

Sincerely yours,
Zheng

**Comment by p-zubieta** [comment]
Hi @zheng001a. Depending on your collective variable for this problem, FFS might not be the best method to compute the free energy. We offer a good collection of sampling methods that allow you to do exactly this (see [here](https://ssagesproject.github.io/docs/Advanced%20Sampling%20Methods.html)). Can you describe in more detail, what collective variable are you using?

**Reply by zheng001a** [reply]
Thanks for your reply.

I using the FFS method to simulate the process of the droplet on the surface of the groove transforming from the Cassie state to the Wenzel state, where I use penetration (the number of particles falling into the groove / the number of particles when the liquid fills the groove) as collective variable, and finally hope to get free energy profile with penetration as order parameter.

I'm trying to get the free energy profile by the mean first passage times method(ref. doi: 10.1063/1.4938248), but that method requires statistics of the time it takes for the trajectory from λA to λi, and the time it takes from λi to return λA. Could you please tell me if that data is available in the software.

I started with the FFS tutorial in /Examples/User/ForwardFlux/LAMMPS/Langevin, but only the N0_total simulation time output into “initial_flux_value.dat” file. 


Sketch of the Cassie (C) and Wenzel (W) wetting states.
 
![image](https://user-images.githubusercontent.com/109740741/182322507-950c617f-9318-42c2-97af-f86ba7e258fc.png)
